### PR TITLE
feat(components/google-cloud): Move flex template python files from piper

### PR DESCRIPTION
**Pull request by xiaowenxie5**
**Description of your changes:**

Moves over relevant files from the dataflow flex template directory. Imports still need to be fixed, but this is the first step to eventually release this publicly.

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: components/google-cloud/google_cloud_pipeline_components/experimental/evaluation/flex/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# A DataFlow Flex Templates compatible container for model evaluation.
+
+# TODO(b/169671349): Restore when 3.8 is in the base image.
+# FROM gcr.io/dataflow-templates-base/python3-template-launcher-base
+FROM gcr.io/dataflow-templates-base/python3-template-launcher-base-staging:flex_templates_base_image_release_20201026_RC00
+
+RUN python --version
+
+WORKDIR /
+ADD google3 /google3
```
[the remark this answers is not in the document]
Comment (IronPan) [reply components/google-cloud/google_cloud_pipeline_components/experimental/evaluation/flex/Dockerfile:10]: a follow up PR is fine.

[the remark this answers is not in the document]
Comment (xiaowenxie5) [reply components/google-cloud/google_cloud_pipeline_components/experimental/evaluation/flex/Dockerfile:10]: Will do in followup!

**Pull request merged**